src/base/utils/Reaction.jl

```diff
@@ -100,3 +100,47 @@ reaction_mass_balanced(model::StandardModel, rxn::Reaction) =
 
 reaction_mass_balanced(model::StandardModel, reaction_dict::Dict{String,Float64}) =
     all(values(reaction_atom_balance(model, reaction_dict)) .== 0)
+
+"""
+    stoichiometry_string(rxn_dict::Dict{String, Float64}; remove_compartment=false)
+
+Return the reaction equation as a string. Removes trailing `_x` where `x` is any character
+if `remove_compartment` is `true`.
+
+# Example
+```
+julia> req = Dict("coa_c" => -1, "for_c" => 1, "accoa_c" => 1, "pyr_c" => -1);
+julia> stoichiometry_string(req)
+"coa_c + pyr_c = for_c + accoa_c"
+```
+"""
+function stoichiometry_string(req; remove_compartment = false)
```

Comment:
this "remove compartment" sounds wrong, it may easily erase whatever is not in there actually. Also what if there's `..._ex` or `..._cy` ?